src/main/java/com/eprosima/fastdds/fastddsgen.java

```diff
@@ -295,6 +299,10 @@ else if (arg.equals("-no-typesupport"))
             {
                 generate_typesupport_ = false;
             }
+            else if (arg.equals("-no-dependencies"))
```

Comment:
Instead of having the same literal string "-no-dependencies" in several places, create a constant
```java
static final String no_dependencies_arg = "-no-dependencies";
```
and use it.

Reply:
This string is only used when the args are checked, the same as the rest of the arguments.
Should I create constants for them to?

Reply:
Created constants in [e556f9f](https://github.com/eProsima/Fast-DDS-Gen/pull/275/commits/e556f9f6356ed00786631eda51387fcbb58a4072)